examples/local_elasticsearch.conf

```diff
@@ -0,0 +1,26 @@
+input {
+  http_poller {
+    urls => {
```

Comment:
@maximede fully understood. Unfortunately, if the request is complex it'll look complex. I think this falls under giving people a reasonable amount of rope to hang themselves with, and providing them with a shorter rope by default.